r/R/dplyr-functions.R

```diff
@@ -808,3 +808,25 @@ agg_funcs$all <- function(x, na.rm = FALSE) {
     options = list(na.rm = na.rm, na.min_count = 0L)
   )
 }
+
+agg_funcs$mean <- function(x, na.rm = FALSE) {
+  list(
+    fun = "mean",
+    data = x,
+    options = list(na.rm = na.rm, na.min_count = 0L)
+  )
+}
+agg_funcs$sd <- function(x, na.rm = FALSE) {
```

Comment:
It doesn't look like you're passing in `na.rm` to the options. If that's because of ARROW-13691, leave a comment here about that.

Reply:
Good point - updated now!